src/index.js

```
@@ -114,21 +114,41 @@ function execProxy(filePath: string, parameters: Array<string> = [], givenOption
   let spawnedProcess = null
   const promise: Object = exec(filePath, parameters, options, function(spawnedChildProcess) {
     if (killSignal) {
-      spawnedChildProcess.kill(killSignal)
+      // eslint-disable-next-line no-use-before-define
+      killProcess(spawnedChildProcess, killSignal)
     } else {
       spawnedProcess = spawnedChildProcess
     }
   })
   promise.kill = function(givenKillSignal) {
     if (spawnedProcess) {
-      spawnedProcess.kill(givenKillSignal)
+      // eslint-disable-next-line no-use-before-define
+      killProcess(spawnedProcess, givenKillSignal)
     } else {
       killSignal = givenKillSignal || 'SIGTERM'
     }
   }
   return promise
 }
 
+function killProcess(spawnedProcess: Object, signal: string = 'SIGTERM'): void {
+  const spawnfile = Path.basename(spawnedProcess.spawnfile)
+  if (process.platform !== 'win32' && spawnfile !== 'wmic.exe' && spawnfile !== 'wmic') {
+    // Also do this if the process is wmic
+    spawnedProcess.kill(signal)
+    return
+  }
+  execProxy('wmic', ['process', 'where', `(ParentProcessId=${spawnedProcess.pid})`, 'get', 'processid'], { stream: 'stdout', timeout: 60 * 1000 })
+    .then(function(output: any) {
+      const pids = output.split(/\s+/).filter(i => /^\d+$/.test(i)).map(parseInt).filter(i => i !== spawnedProcess.pid && i > 0)
+      pids.forEach(function(pid) {
+        process.kill(pid, signal)
```

Atom ignores errors in attempting to kill these child processes, while here we are propagating them up to the `catch` on L147.